src/Impostor.Server/Net/Inner/Objects/Components/InnerPlayerPhysics.cs

```diff
@@ -27,12 +27,12 @@ public InnerPlayerPhysics(ICustomMessageManager<ICustomRpc> customMessageManager
 
         public override ValueTask<bool> SerializeAsync(IMessageWriter writer, bool initialState)
         {
-            throw new NotImplementedException();
+            return new ValueTask<bool>(false);
```

Comment:
Please keep it the way it was, Impostor rarely serializes network objects (InnerPlayerInfo is the exception here) so throwing when someone attempts to serialize this object makes sense. Other classes like InnerLobbyBehavior, InnerPlayerControl, InnerMeetingHud also throw on serialize.

Reply:
I think you should use Logger to prompt it instead of throwing the